src/config/dark-sites.config

```diff
@@ -15,6 +15,7 @@
 *.sapph.xyz
 *.starbreeze.com
 *.suyu.dev
+*.thesigmaarts.com
```

Comment:
Do you use prefixes for the site address for other sub-pages? I could not find any like that using links on your site.

As I do not saw anything with prefix I need to asc to add this page without `*.` in the correct order.

Reply:
`furaffinity.net` is bad example - it has sites with prefixes like `https://shop.furaffinity.net/`

I went through all links of `thesigmaarts.com` and all other sites are open as suffix `https://thesigmaarts.com/shop`

and for that `*.` prefix is not needed in dark-sites

I just want to have clear entry. Just don't want everyone adding *.mysite.isdark with asterisks because they are lazy.

You change `*.thesigmaarts.com` to `thesigmaarts.com` and I will merge.